cpp/CMakeLists.txt

```diff
@@ -28,9 +28,9 @@ include(FetchContent)
 FetchContent_Declare(
   iree
   GIT_REPOSITORY https://github.com/iree-org/iree.git
-  GIT_TAG 7ca0f46fc38c7d5ccb7ee275db59ed7d992436ef # 2022-06-09
+  GIT_TAG c4624870a8de85224c3f02f4ea8e26837cd99e2f # 2022-08-12
   GIT_SUBMODULES_RECURSE OFF
-  GIT_SHALLOW ON
+  GIT_SHALLOW OFF
```

Comment:
Why this change?

Reply:
CMake wasn't actually fetching the specified commit without it. It might work for the tip of a branch, but I have this pinned to an older commit without a specific branch.